### DISCUSSION TITLE
design windows not appear

### DISCUSSION THREAD

**Discussion by shathulan** [eclipse/windowbuilder#656] [Q&A]
i am doing my work in office pc and i am coping that folder to my usb, after i am trying to open that folder where codes are working but design menu not appearing . i did java swing with eclipse.

**Comment by ptziegler** [comment]
Is there an error message when you try to open the design page?

Are there any compilation errors in your workspace that would prevent compilation of the source code?

Could you perhaps upload a screenshot of your IDE that shows the issue? Because I have a hard time visualizing the problem with just your description alone.

**Reply by martie12** [reply]
![image](https://github.com/eclipse-windowbuilder/windowbuilder/assets/143509679/40d098db-6c4f-471c-9b11-1cfe54c21bce)

**Reply by martie12** [reply]
You know what should i do?

**Reply by ptziegler** [reply]
That looks a lot like the WindowBuilder installation is broken. If you only need Swing support, then you don't need the XWT component, so my suggestion is to simply uninstall it.

In the menu bar, click Help -> About Eclipse IDE -> Installation Details

In the Installed Software tab, select both "WindowBuilder SWT Designer XWT Support" and "WindowBuilder XML Core" and click "Uninstall...". That might already be enough to fix this problem.

![image](https://github.com/eclipse-windowbuilder/windowbuilder/assets/70652978/a59bed7e-f3ef-4cdb-8d28-98b904999eb4)

**Reply by martie12** [reply]
Thank you!

**Reply by achrafAmesGuine** [reply]
It works smoothly for me, Thanks a lot!

**Comment by shathulan** [comment]
i am already install windowbuilder this t
is did from my office pc after i export as archive file and in my personal
lap i am import that file. in meantime this form *design **window*s does
not appear.
[image: image.png]

**Comment by shathulan** [comment]
[image: Screenshot 2023-12-12 222428.png]
i am uninstalled but it seems not appear

**Reply by ptziegler** [reply]
None of the screenshots are uploaded to GitHub, so I can only speculate what the error is.

Does the design page not appear at all or does open with the "Internal Error" message shown in the initial response?

**Reply by FancyFence3** [reply]
I'm having this exact error. It says internal error for me. I tried uninstalling that file, but it doesn't seem to be working

**Reply by ptziegler** [reply]
@FancyFence3 

Can you post a screenshot of your "Installed Plugins" tab, with the filter set to "org.eclipse.wb"?

![image](https://github.com/eclipse-windowbuilder/windowbuilder/assets/70652978/8a44cab6-a410-43f1-b7f1-5bd89b873590)

**Reply by FancyFence3** [reply]
Sure, this is what shows up
![image](https://github.com/eclipse-windowbuilder/windowbuilder/assets/153991288/b51b47e9-c28f-4134-b209-5db2928f318e)

**Reply by ptziegler** [reply]
Thanks, that already helps :)

> org.eclipse.wb.xwt 1.9.0.2016082504...

This is an _ancient_ version of the XWT plugin. No wonder this doesn't work.

Can you also post a screenshot of the "Installed Software" tab (again filtered for org.eclipse.wb)?

**Reply by FancyFence3** [reply]
it might be outdated because of some leftover files or something, from when I tried installing it using my school's tutorial, which I later found out was very outdated
![image](https://github.com/eclipse-windowbuilder/windowbuilder/assets/153991288/18a0223d-5fee-4ac4-ba79-5a1e6a93382a)
The filter for org.eclipse.wb wasn't working for some reason, but I think these are the files of WindowBuilder that I have

**Reply by ptziegler** [reply]
Yeah... that doesn't look right.

Can you manually uninstall the `CSS Support` and the `XML Core` feature? Those are leftovers from a previous WindowBuilder installation.

**Reply by FancyFence3** [reply]
Alright, they are gone, what do I need to do next?

**Reply by ptziegler** [reply]
In principle, you only have to restart the Eclipse afterwards.

After you've done that, can you check whether the XWT bundle is also gone from the plugins tab? If so, then I'd expect that the editor should work again.

**Reply by FancyFence3** [reply]
The outdated files are gone from the Installed Software tab, however they are still in the Plugins tab and I'm still getting the error, even after restarting. I don't know how to get rid of them

**Reply by ptziegler** [reply]
Hm... now that makes things a little bit more difficult.

As a last-ditch effort, you could try going through the previous configurations in the "Installation History" tab and revert to a configuration _before_ you've installed WindowBuilder 1.9.0 and then do a clean installation of the latest release, afterwards.

![image](https://github.com/eclipse-windowbuilder/windowbuilder/assets/70652978/93a83f66-a117-4a27-80ea-b931dba74bb2)

If that still doesn't work, I'm almost tempted to suggest setting up a fresh Eclipse IDE. You can import any previously installed features via the import wizard in `File -> Import`

In the dialog that is opened by  "From Existing Installation", you can choose your current Eclipse installation and then pick everything (except WindowBuilder of course) that needs to be carried over.

![image](https://github.com/eclipse-windowbuilder/windowbuilder/assets/70652978/2b096b20-bb2c-4c3f-a954-216fe93504a4)

**Reply by FancyFence3** [reply]
I tried reinstalling eclipse and the plugin, and it finally works! Thank you very much for the time you took to help :)
